challenge.js

```diff
@@ -43,6 +47,8 @@
 
 // ANSWER
 
+  /** document.querySelector('p') **/
```

Comment:
That CSS selector will absolutely work. It'll get you every p element, including E. `document.querySelector`, though, returns only the first element it finds that matches the selector; this may not be E. `document.querySelectorAll` will retrieve the collection of elements matched by the selector string.

The question was a set-up to use the hierarchy of the page to your advantage. Here is an example:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.

3/4